client/interfaces.go

```diff
@@ -61,7 +61,7 @@ type ImageServer interface {
 	GetImageAliases() (aliases []api.ImageAliasesEntry, err error)
 	GetImageAliasNames() (names []string, err error)
 
-	GetImageAlias(name string) (alias *api.ImageAliasesEntry, ETag string, err error)
```

Comment:
The commit message only mentions `lxc` but this commit changes many modules. What do you think of breaking this commit in parts for `client`, `lxc` and `lxd-benchmark`?

Reply:
I had some problems doing these rebases before. So if you do this and don't know how, I suggest doing an interactive rebase, set the previous commit as edit, cherry pick this commit with -`-no-commit` and commit the changes separately. Just say the word if you want a hand!

Reply:
yes agree, this should be several commits, at least 1 per package to aid with backporting.